hazelcast/src/main/java/com/hazelcast/internal/bootstrap/TpcServerBootstrap.java

```diff
@@ -63,19 +68,31 @@ public class TpcServerBootstrap {
     private final boolean enabled;
     private final Map<Eventloop, Supplier<? extends ReadHandler>> readHandlerSuppliers = new HashMap<>();
     private final List<AsyncServerSocket> serverSockets = new ArrayList<>();
+    private final Config config;
     private volatile List<Integer> clientPorts;
 
     public TpcServerBootstrap(NodeEngineImpl nodeEngine) {
         this.nodeEngine = nodeEngine;
         this.logger = nodeEngine.getLogger(TpcServerBootstrap.class);
         this.ss = (InternalSerializationService) nodeEngine.getSerializationService();
-        this.enabled = Boolean.parseBoolean(getProperty("hazelcast.tpc.enabled", "false"))
-                || nodeEngine.getConfig().getAltoConfig().isEnabled();
-        logger.info("TPC: " + (enabled ? "enabled" : "disabled"));
+        this.config = nodeEngine.getConfig();
+        this.enabled = enabled();
         this.thisAddress = nodeEngine.getThisAddress();
         this.tpcEngine = newTpcEngine();
     }
 
+    private boolean enabled() {
```

Comment:
I would call this something like 'loadAltoEnabled'.

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/7b04dacce8b8cd9655d863a8d37dd13774babf06